Cargo.toml

```
@@ -39,14 +39,10 @@ esp8266 = []
 # communication method (note that "uart" is enabled by default):
 uart        = []
 jtag_serial = ["portable-atomic"] # C3, C6, H2, and S3 only!
```

Since we're breaking stuff anyway, and this has been bothering for awhile now... If you wouldn't mind:

```suggestion
jtag-serial = ["portable-atomic"] # C3, C6, H2, and S3 only!
```

---

yes please, thanks!